virtualcluster/pkg/syncer/constants/constants.go

```diff
@@ -64,6 +64,9 @@ const (
 	// LabelSecretUID is the service account token secret UID in tenant namespace.
 	LabelSecretUID = "tenancy.x-k8s.io/secret.UID" // #nosec G101 -- This is a label key
 
+	// LabelTenantSchedulerName is used to by resources that do not need to be synced, but to be scheduled to a tenant controlled node
+	LabelTenantSchedulerName = "tenancy.x-k8s.io/tenant-scheduler"
```

Comment:
I need to use this label to prevent other resources being synced to super cluster in next step. This is why I did not pick label name specific to "pod"....